Rakefile

```diff
@@ -717,29 +717,33 @@ You can open Terminal by:
 
 ## Before you start...
 
-Today, you will be installing the basic software you need for the class.
+PLEASE READ THIS WHOLE PAGE! You're going to be installing a lot of stuff on your computer in a way you probably never have before.
+
+- THE WAY THIS WORKS: This script is going to check for different pieces of software on your computer. If you don't have those pieces of software -- or you do, but they need to be tweaked -- the script will stop and tell you what to do. When you're done, the script will restart. This will continue until everything is set up correctly.
+
+- You should be able to COPY AND PASTE the lines into Terminal. We don't recommend typing the commands manually since a single typo can make a command malfunction.
+
+- If you see words in all-capitals (like "YOUR NAME"), they're a placeholder that you should replace with whatever's appropriate.
 
-If you have ANY questions, raise your hand for assistance.
+- Copy and paste ONE COMMAND AT A TIME. Each command should be a single line, but note that if your window is narrow some lines may "wrap" and appear as multiple lines. Most commands will begin with a dollar sign (`$`), so keep an eye out for that. On that note...
 
-1. You will be reading instructions from one Terminal window and
-2. Entering these commands into *another* Terminal window.  We recommend you arrange them side-by-side.
-** If you don't have the additional Terminal open, do so now. **
+- You **SHOULD NOT** include the `$` when copying and pasting -- include everything AFTER it. The dollar sign is an industry convention that signals "This is an individual command to be copied and pasted."
 
+- You should have TWO TERMINAL WINDOWS open at the same time: one in which this script is running, and one in which you can type stuff. This is because you can't type stuff into the window in which this script is running -- you can only hit the "return" key when prompted -- so you'll need the other window to actually issue commands. Open a new window by pressing Command + N.
 
-Each package will list the installation steps.
+## Getting started...
+
+If you haven't done so already, open the Terminal app so you can begin entering commands.
+
+You can open Terminal by:
 
-Read through every instruction carefully and follow them to the letter.
+- Typing "Terminal" into Spotlight (ensure you select the Terminal app). Open spotlight by clicking the little magnifying glass in the top-right corner of your screen, or hitting Command + Space.
 
-You should be able to copy and paste the lines into Terminal.
-With these exceptions:
-- If you see all capitals (like "YOUR NAME"), replace this placeholder with your appropriate information.
-- Commands will start with a dollar sign (`$`),
-but you should NOT actually copy/type the `$` into your Terminal.
-The dollar sign is a standard convention to indicate a bash (terminal) command.
+- In the Finder, going to the "Applications" folder, then "Utilities", then double-clicking "Terminal".
```

Comment:
I think we should follow this with some discussion of what directory they should be in, in this second terminal.  "You do not need to change to the ~/wdi directory.  All commands can be run from the default "home" directory.